### PR TITLE
Switch order of padding and resizing in eval

### PR DESCRIPTION
**Pull request by karakusc**
Fixes a bug in bs=2. Resizing should not happen on the padded image.

Convergence results:
0411 02:42:57 @monitor.py:467] mAP(bbox)/IoU=0.5: 0.59276
[0411 02:42:57 @monitor.py:467] mAP(bbox)/IoU=0.5:0.95: 0.37485
[0411 02:42:57 @monitor.py:467] mAP(bbox)/IoU=0.75: 0.40636
[0411 02:42:57 @monitor.py:467] mAP(bbox)/large: 0.48575
[0411 02:42:57 @monitor.py:467] mAP(bbox)/medium: 0.41108
[0411 02:42:57 @monitor.py:467] mAP(bbox)/small: 0.22027
[0411 02:42:57 @monitor.py:467] mAP(segm)/IoU=0.5: 0.56142
[0411 02:42:57 @monitor.py:467] mAP(segm)/IoU=0.5:0.95: 0.34545
[0411 02:42:57 @monitor.py:467] mAP(segm)/IoU=0.75: 0.36453
[0411 02:42:57 @monitor.py:467] mAP(segm)/large: 0.47087
[0411 02:42:57 @monitor.py:467] mAP(segm)/medium: 0.37721
[0411 02:42:57 @monitor.py:467] mAP(segm)/small: 0.18613

### CODE REVIEW

File: MaskRCNN/eval.py [hunk 1]
```diff
@@ -143,11 +144,11 @@ def predict_image_batch(img_batch, model_func, orig_sizes):
             boxes[ind, :] = boxes[ind, :]/scales[i]
 
             # boxes are already clipped inside the graph, but after the floating point scaling, this may not be true any more.
-            boxes[ind, :] = clip_boxes(boxes[ind, :], orig_sizes[i])
+            boxes[ind, :] = clip_boxes(boxes[ind, :], org_sizes[i])
```
Comment (armandmcqueen) [inline MaskRCNN/eval.py]: Any reason to `org_sizes` over `orig_sizes`? I think the latter is slightly clearer

File: MaskRCNN/eval.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def predict_image(img, model_func):
     results = [DetectionResult(*args) for args in zip(boxes, probs, labels, masks)]
     return results
 
-def predict_image_batch(img_batch, model_func, orig_sizes):
+def predict_image_batch(img_batch, model_func, resized_sizes, scales, org_sizes):
```
Comment (armandmcqueen) [inline MaskRCNN/eval.py]: Can we remove the `scales` argument? I don't see it being used

**Comment by armandmcqueen** [comment]
I prefer `orig_sizes` to `org_sizes`, but the PR looks good. Approved

File: MaskRCNN/eval.py (hunk 1 above)
Comment (karakusc) [reply MaskRCNN/eval.py]: Fixed, `org_sizes` was legacy from an intermediate state of the code when both `orig_sizes` and `org_sizes` existed.

**Pull request merged**